starter-core/src/main/java/io/micronaut/starter/feature/coherence/CoherenceFeature.java

```diff
@@ -32,11 +32,11 @@
 @Singleton
 public class CoherenceFeature implements Feature {
 
-    public static final String COHERENCE_VERSION = "21.06.2";
```

Comment:
Previously we were adding this version to `gradle.properties`, and then using that as the version in the Gradle and Maven builds.

Now the Coherence module has a BOM, I believe this is the wrong behaviour (and it would require changes to the kotlin dsl scripts to import the property)

I've removed this from here, and the version of the `com.oracle.coherence.ce` artifacts instead comes from the coherence bom